### ISSUE TITLE
Error messages when saving an image

### ISSUE THREAD

**Issue by svniemeijer** [stcorp/visan#3]
When I try to save an image from the World Plot or 2D Plot window, it is not possible to change the directory. Clicking on ‘Browse…’ gives the following error message:

```
Traceback (most recent call last):
File “D:\lenka-hp\soft\Anaconda3\envs\visan_env\lib\site-packages\visan\plot\typedsavefiledialog.py”, line 217, in OnBrowse
filename = wx.FileSelector(“Save As…”, default_path=directory, flags=wx.SAVE, parent=self)
AttributeError: module ‘wx’ has no attribute ‘SAVE’
```

**Comment by svniemeijer** [comment]
Fixed in 5301fe7f6a8d9121d46864156724a0e0ce821ed0

**Comment by OPTICSFOOL** [comment]
I also have this problem, and I want to know how to solve it.
